### ISSUE TITLE
Nothing happens

### ISSUE THREAD

**Issue by kvq** [Genymobile/gnirehtet#281]
`2020-03-12 19:36:10.120 I Gnirehtet: Checking gnirehtet client...
2020-03-12 19:36:10.120 I Gnirehtet: Starting relay server on port 31416...
2020-03-12 19:36:10.606 I Relay: Relay server started`
Nothing happens after that.

Android 10, app installed, debuggio ng is on. NVPN popup, app doesn't seem to boot.

**Comment by rom1v** [comment]
Which version? What command do you type?

**Comment by rom1v** [comment]
Is your device detected when you type `adb devices`?

**Comment by kvq** [comment]
> Is your device detected when you type `adb devices`?

Yeah, it does appear in devices and application installs every time with no problem.

**Comment by RadoYankov** [comment]
same here. Android 10, app is installed, no VPN popup.

**Comment by rom1v** [comment]
When you start it, any logs in `adb logcat`?

**Comment by RadoYankov** [comment]
no logs related to it, no.

**Comment by rom1v** [comment]
> my problem is with the java version. Tried the rust one, everything seems to work fine.

I'm surprised, they should do the exact same thing (and I cannot reproduce).

Could you try again the java version and confirm that it does not work, while the rust version works, please?

**Comment by RadoYankov** [comment]
yea you're right. The java one works now. Not sure what happened there.
